src/browser-plugins/playwright-plugin.js

```diff
@@ -24,7 +24,7 @@ class PlaywrightPlugin extends BrowserPlugin {
         } else {
             browser = await this.library.launchPersistentContext(userDataDir, launchOptions);
         }
-
+        // @TODO: Rework the disconnected events once the browser context vs browser issue is fixed
```

Comment:
Will this be done in this PR or other PR?

Reply:
It will be done in the PlaywrightBrowser pr.